docs/source/conf.py

```diff
@@ -171,18 +168,24 @@ def setup(app):
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'nabs', 'nabs Documentation',
-     author, 'nabs', 'One line description of project.',
-     'Miscellaneous'),
+    (
+        master_doc,
+        "nabs",
+        "nabs Documentation",
+        author,
+        "nabs",
+        "One line description of project.",
+        "Miscellaneous",
+    ),
 ]
 
 
 # -- Extension configuration -------------------------------------------------
 intersphinx_mapping = {
-    'bluesky': ('https://blueskyproject.io/bluesky', None),
-    'numpy': ('https://docs.scipy.org/doc/numpy', None),
-    'ophyd': ('https://blueskyproject.io/ophyd', None),
-    'pcdsdaq': ('https://pcdshub.github.io/pcdsdaq', None),
-    'pcdsdevices': ('https://pcdshub.github.io/pcdsdevices', None),
-    'python': ('https://docs.python.org/3', None),
+    "bluesky": ("https://blueskyproject.io/bluesky", None),
+    "numpy": ("https://docs.scipy.org/doc/numpy", None),
+    "ophyd": ("https://blueskyproject.io/ophyd", None),
+    "pcdsdaq": ("https://pcdshub.github.io/pcdsdaq", None),
+    "pcdsdevices": ("https://pcdshub.github.io/pcdsdevices", None),
```

Comment:
Interesting that this intersphinx section didn't cause problems as it did in typhos.  https://github.com/pcdshub/typhos/blob/6032b9b8c113032af4924731b5e8cec03f4f04eb/docs/source/conf.py#L198

Build error here: https://github.com/pcdshub/typhos/actions/runs/4177086683/jobs/7234218014#step:14:26

I had to specify a tag for pcdsdevices since the unspecified (https://pcdshub.github.io/pcdsdevices) is a redirect